### PR TITLE
Initial Customer and User model.

### PR DESCRIPTION
**Pull request by blentz**
It is a first step at implementing Customer on-boarding (#93), but does not yet fully satisfy the request. More work is needed before the feature is complete.

### CODE REVIEW

**Comment by blentz** [comment]
I've stripped out the UserPreference bits, so now it's just the Customer API, with the dependent User model.

File: koku/api/iam/view/customer.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from rest_framework import permissions, viewsets
+
+import api.iam.model as model
+import api.iam.serializers as serializers
+
+class CustomerViewSet(viewsets.ModelViewSet):
+    """ generic customer list """
+    queryset = model.Customer.objects.all()
+    serializer_class = serializers.CustomerSerializer
+    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
```
[the remark this answers is not in the document]
Comment (blentz) [reply koku/api/iam/view/customer.py]: Sure, I'm okay removing that.

File: koku/api/iam/view/customer.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+"""View for Customers."""
+
+from rest_framework import permissions, viewsets
+
+import api.iam.model as model
+import api.iam.serializers as serializers
+
+class CustomerViewSet(viewsets.ModelViewSet):
+    """ generic customer list """
```
[the remark this answers is not in the document]
Comment (blentz) [reply koku/api/iam/view/customer.py]: It sounds like we'll need to support delete, we just need to explicitly think about what it means.

We also need update for making changes, e.g. changing the Customer Name. But we may need additional stories to fully flesh out the flow.

[the remark this answers is not in the document]
Comment (chambridge) [reply koku/api/iam/view/customer.py]: That is fine.

**Pull request merged**